agent/engine/serviceconnect/manager_other.go

```diff
@@ -69,3 +69,15 @@ func (*manager) GetLoadedImageName() (string, error) {
 		"appnetAgent container get image name: unsupported platform: %s/%s",
 		runtime.GOOS, runtime.GOARCH))
 }
+
+func (*manager) GetLoadedAppnetVersion() (string, error) {
+	return "", loader.NewUnsupportedPlatformError(fmt.Errorf(
+		"appnetAgent container get loaded appnet version: unsupported platform: %s/%s",
```

Comment:
Can we update the log message to `AppNet agent` for line 75 and line 81 in the follow up PR? Thanks!

Reply:
Was following the same convention as other functions in this file...